presto-product-tests/src/main/java/io/prestosql/tests/hive/TestExternalHiveTable.java

```diff
@@ -46,6 +53,27 @@ public Requirement getRequirements(Configuration configuration)
                 mutableTable(NATION_PARTITIONED_BY_BIGINT_REGIONKEY));
     }
 
+    @Test
+    public void testCreateExternalTableWithInaccessibleSchemaLocation()
+            throws IOException
+    {
+        File tempDir = createTempDir();
```

Comment:
The temp dir is local in the tests' containers, so it can as well not exist at all, am i right?

Reply:
Hmmmm, good point, not sure if this step is even needed. I'll need to verify though.